src/loaders/eventLoader.js

```diff
@@ -217,4 +217,8 @@ const getFilterOptionNames = async (filters, headers) => {
     );
 };
 
+// Empty filter sometimes returned for saved maps
+const isValidDimension = ({ dimension, items }) =>
+    Boolean(dimension && !(items && !items.length));
```

Comment:
Is `{dimension: 'something', items: null}` valid?  If so, maybe this would be a little cleaner without the double negation as `dimension && (!item || items.length)` (though maybe not any better, they're both a little hard to read) and if not then it should be `dimension && items && items.length`

Reply:
Changed to more readable `dimension && (!item || items.length)` :-)